File: SDKs/Java/src/main/java/com/aspose/words/model/PageNumber.java
package com.aspose.words.model;

public class PageNumber {
  private String Format = null;
  private String Alignment = null;
  private Boolean IsTop = null;
  private Boolean SetPageNumberOnFirstPage = null;
  /**
	 * getFormat
	 * Gets String
	 * @return Format
	 */
  public String getFormat() {
    return Format;
  }

	/**
	 * setFormat
	 * Sets String
	 * @param Format String
	 */
  public void setFormat(String Format) {
    this.Format = Format;
  }

  /**
	 * getAlignment
	 * Gets String
	 * @return Alignment
	 */
  public String getAlignment() {
    return Alignment;
  }

	/**
	 * setAlignment
	 * Sets String
	 * @param Alignment String
	 */
  public void setAlignment(String Alignment) {
    this.Alignment = Alignment;
  }

  /**
	 * getIsTop
	 * Gets Boolean
	 * @return IsTop
	 */
  public Boolean getIsTop() {
    return IsTop;
  }

	/**
	 * setIsTop
	 * Sets Boolean
	 * @param IsTop Boolean
	 */
  public void setIsTop(Boolean IsTop) {
    this.IsTop = IsTop;
  }

  /**
	 * getSetPageNumberOnFirstPage
	 * Gets Boolean
	 * @return SetPageNumberOnFirstPage
	 */
  public Boolean getSetPageNumberOnFirstPage() {
    return SetPageNumberOnFirstPage;
  }

	/**
	 * setSetPageNumberOnFirstPage
	 * Sets Boolean
	 * @param SetPageNumberOnFirstPage Boolean
	 */
  public void setSetPageNumberOnFirstPage(Boolean SetPageNumberOnFirstPage) {
    this.SetPageNumberOnFirstPage = SetPageNumberOnFirstPage;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class PageNumber {\n");
    sb.append("  Format: ").append(Format).append("\n");
    sb.append("  Alignment: ").append(Alignment).append("\n");
    sb.append("  IsTop: ").append(IsTop).append("\n");
    sb.append("  SetPageNumberOnFirstPage: ").append(SetPageNumberOnFirstPage).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/PageNumber.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class PageNumber {
    public string Format { get; set; }

    public string Alignment { get; set; }

    public bool IsTop { get; set; }

    public bool SetPageNumberOnFirstPage { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class PageNumber {\n");
      sb.Append("  Format: ").Append(Format).Append("\n");
      sb.Append("  Alignment: ").Append(Alignment).Append("\n");
      sb.Append("  IsTop: ").Append(IsTop).Append("\n");
      sb.Append("  SetPageNumberOnFirstPage: ").Append(SetPageNumberOnFirstPage).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
